src/components/Placeholder/index.tsx

```diff
@@ -94,7 +100,11 @@ Placeholder.propTypes = {
 }
 
 Placeholder.defaultProps = {
+  col: undefined,
```

Comment:
Why ? 🤔

Reply:
To satisfy linter on commit :/

Reply:
ie. 

95:3   error    propType "col" is not required, but has no corresponding defaultProps declaration     react/require-default-props
  96:3   error    propType "height" is not required, but has no corresponding defaultProps declaration  react/require-default-props
  97:3   error    propType "length" is not required, but has no corresponding defaultProps declaration  react/require-default-props
  99:3   error    propType "width" is not required, but has no corresponding defaultProps declaration   react/require-default-props

Reply:
Oh wait. I need to remove defaultProps completely 😂